openfisca_us/parameters/benefit/SNAP/amounts/household_size.yaml

```diff
@@ -0,0 +1,40 @@
+description:  Maximum monthly SNAP benefit by household size
+
+1:
+  values:
+    2021-01-01: 204
```

Comment:
Per the changes scheduled for October (see #27), verify if these should be `2020-10-01` instead